app/services/invite_schools.rb

```diff
@@ -90,21 +78,40 @@ def invite_to_beta(school_urns)
         recipient: induction_coordinator.email,
         name: induction_coordinator.full_name,
         school_name: school.name,
-        start_url: start_url,
+        start_url: private_beta_start_url,
       ).deliver_later
     rescue StandardError
       logger.info "Error emailing school, urn: #{urn} ... skipping"
     end
   end
 
+  def send_beta_chasers
+    beta_school_ids = FeatureFlag.new(name: :induction_tutor_manage_participants).feature.selected_objects.pluck(:object_id)
+    beta_schools = School.where(id: beta_school_ids)
+    beta_schools_without_participants = beta_schools.left_outer_joins(:early_career_teacher_profiles)
+                                                    .left_outer_joins(:mentor_profiles)
+                                                    .where(
+                                                      early_career_teacher_profiles: { id: nil },
+                                                      mentor_profiles: { id: nil },
+                                                    )
+
+    beta_schools_without_participants.each do |school|
+      induction_coordinator = school.induction_coordinators.first
```

Comment:
will that always be non-nil?

Reply:
(by `this` I mean induction coordinator whose email we use later)

Reply:
There's an edge case where we invite a school to beta then remove their induction coordinator. I'm happy for that to fail though